libraries/networking/src/webrtc/WebRTCSocket.h

```diff
@@ -0,0 +1,161 @@
+//
+//  WebRTCSocket.h
+//  libraries/networking/src/webrtc
+//
+//  Created by David Rowe on 21 Jun 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+
+#ifndef vircadia_WebRTCSocket_h
+#define vircadia_WebRTCSocket_h
+
+#include <shared/WebRTC.h>
+
+#if defined(WEBRTC_DATA_CHANNELS)
+
+#include <QAbstractSocket>
+#include <QObject>
+#include <QQueue>
+
+#include "WebRTCDataChannels.h"
+#include "WebRTCSignalingServer.h"
+
+/// @addtogroup Networking
+/// @{
+
+
+/// @brief Provides a QUdpSocket-style interface for using WebRTCDataChannels.
+class WebRTCSocket : public QObject {
```

Comment:
Perhaps it should inherit from QAbstractSocket?

Reply:
Ditto.